src/os/config/displaysettings.js

```diff
@@ -522,3 +533,21 @@ os.config.DisplaySettingsCtrl.prototype.updateSunlight = function() {
 os.config.DisplaySettingsCtrl.prototype.updateTerrain = function() {
   this.updateSetting_(os.config.DisplaySetting.ENABLE_TERRAIN, this['terrainEnabled']);
 };
+
+
+/**
+ * Update the max feature count.
+ * @param {number} update3D Whether to update 3D or 2D.
```

Comment:
`boolean`?

Reply:
Woops. Originally had this passing the new count as the argument.